.github/workflows/ci.yml

```diff
@@ -43,6 +43,7 @@ jobs:
         run: go test -coverprofile=coverage.txt -v -race ./...
 
       - name: Upload coverage reports to Codecov
+        if: github.actor != 'Hetzner Cloud Bot' && github.actor != 'renovate[bot]'
```

Comment:
Why not target the branch names? Should be more robust.

Reply:
Depending on the context, you'll need different variables. Maybe this is enough: `github.head_ref || github.ref_name`